### PR TITLE
[Trivia] Update entertainment.txt

### PR DESCRIPTION
**Pull request by ghost**
Adding additional correct answers (such as not requiring 'The', and numbers in digits)

Also made one question more specific.

### Type

- [ ] Bugfix
- [x] Enhancement
- [ ] New feature

### CODE REVIEW

**Comment by MiniJennJenn** [comment]
ah sometimes special symbols get messed up in updates. thanks for catching these.

**Comment by MiniJennJenn** [comment]
I don't have any perms in the v2 repo but I'm approving this for when someone comes along to merge this.

**Review by MiniJennJenn — APPROVED** [review]

File: data/trivia/entertainment.txt [hunk 1]
```diff
@@ -530,7 +530,7 @@ How old was Shirley Temple when she made her last film?`21
 Who is Declan Patrick McManus better known as?`Elvis
 Where did Mighty Mouse get his superpowers?`Supermarket
 Who broke Batman?`Bane
-On what LP Cover can we read the words 'Welcome Rolling Stones'?`Sergeant Pepper�s Lonely Hearts Club Band
+On what LP Cover can we read the words 'Welcome Rolling Stones'?`Sergeant Pepper´s Lonely Hearts Club Band
```
Comment (Tobotimus) [inline data/trivia/entertainment.txt]: This `´` character should be an apostrophe `'`

**Review by Tobotimus — CHANGES_REQUESTED** [review]

File: data/trivia/entertainment.txt (hunk 1 above)
Comment (MiniJennJenn) [reply data/trivia/entertainment.txt]: good catch @_@

**Comment by Twentysix26** [comment]
Added in #1268

**Pull request closed without merging**